src/diagnostics/riscv64/disasm-riscv64.cc

```diff
@@ -1374,11 +1483,17 @@ int Decoder::InstructionDecode(byte* instr_ptr) {
     case Instruction::kJType:
       DecodeJType(instr);
       break;
+    case Instruction::kCRType:
+      DecodeCRType(instr);
+      break;
+    case Instruction::kCIType:
+      DecodeCIType(instr);
+      break;
     default:
       Format(instr, "UNSUPPORTED");
       UNSUPPORTED_RISCV();
   }
-  return kInstrSize;
```

Comment:
Make sure we replace all the places where `kInstrSize` is used by `instr->InstructionSize()`; or replace `kInstrSizeLog2` accordingly too